src/main/java/de/zalando/zmon/dataservice/data/KairosDBStore.java

```diff
@@ -230,6 +234,28 @@ void store(WorkerResult wr) {
                 LOG.info("KairosDB Query: {}", query);
             }
 
+            // Store datapoints query!
+            storeDatapoints(wr, query);
```

Comment:
if KairosDB goes down - it takes Redis path with it. Do we want to send data to Redis when Kairos fails?